Measurements/Measurements.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BatteryMonitoring
+{
+    public class Measurements
+    {
+        public static bool IsReadingsAvailable(List<int> readings)
+        {
+            return readings.Count > 0;
+        }
+
+        public static Dictionary<string,int> GetReadingRange(List<int> readings)
+        {
+            Dictionary<string, int> expectedRanges = new Dictionary<string, int>();
+            if (!IsReadingsAvailable(readings))
+            {
+                return expectedRanges;
+            }
```

Comment:
where is the implementation  ?

Reply:
hi sir thanks for the feedback . it is implemented.